tests/istio.mk

```diff
@@ -83,35 +83,82 @@ e2e_all: | $(JUNIT_REPORT)
 	$(MAKE) e2e_simple e2e_mixer e2e_bookinfo \
 	|& tee >($(JUNIT_REPORT) > $(JUNIT_E2E_XML))
 
+# Run the e2e tests, with auth enabled. A separate target is used for non-auth.
 e2e_pilot: istioctl generate_yaml
 	go test -v -timeout 20m ./tests/e2e/tests/pilot ${TESTOPTS} -hub ${HUB} -tag ${TAG}
 
+e2e_pilot_noauth: istioctl generate_yaml
+	go test -v -timeout 20m ./tests/e2e/tests/pilot ${E2E_ARGS} -hub ${HUB} -tag ${TAG} --skip-cleanup -mixer=true -auth=disable -use-sidecar-injector=false
```

Comment:
please remove `--skip-cleanup` from target

why no sidecar injector?

Reply:
It is quite important:
- in minikube, the image can be wiped
- in dev mode - you may want to reuse or test or run individual checks manually after the test is run.

That's what (and how) we've been testing in circle - I just moved them here so it's easier to reproduce, and they can be used by developers. 

I found that every time the test fails - or even if it works - it's good to have the environment for
additional work. There is already a target that doesn't do this.